### ISSUE TITLE
how to disable left when closing right?

### ISSUE THREAD

**Issue by Jagadeesh-Govindaraj** [jshanson7/react-native-swipeable#28]
this is my proplem
1 ,when close the right button its closed. (as expected)
2 , but its activated left button i mean its opened (but i need to disable)

how can i achieve this

**Comment by developer239** [comment]
Yea I am having the same problem. This plugin is unusable on real device.

**Comment by geminiyellow** [comment]
yep, i checked the proptypes, but there is no `disable/enable` flag. 

@jshanson7
